core/src/main/java/org/elasticsearch/action/bulk/BulkItemResponse.java

```diff
@@ -71,12 +83,77 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         return builder;
     }
 
-    static final class Fields {
-        static final String _INDEX = "_index";
-        static final String _TYPE = "_type";
-        static final String _ID = "_id";
-        static final String STATUS = "status";
-        static final String ERROR = "error";
+    public static BulkItemResponse fromXContent(XContentParser parser, int id) throws IOException {
```

Comment:
Can you add a comment explaining that the id? As far as I see it its the bulk item ordering number but would be good to have this info around for future reference.